Tests/TensorFlowTests/OperatorTests/BasicTests.swift

```diff
@@ -26,6 +26,12 @@ precedencegroup StridedRangeFormationPrecedence {
 }
 
 final class BasicOperatorTests: XCTestCase {
+    func testGathering() {
+        let x = Tensor<Float>([[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]])
+        let y = x.gathering(atIndices: Tensor<Int32>(2), alongAxis: 1)
+        assertEqual(y, Tensor<Float>([3.0, 6.0]))
```

Comment:
```suggestion
        XCTAssertEqual(y, Tensor<Float>([3.0, 6.0]))
```

Reply:
I wonder how this passed tests. Where is `assertEqual` defined?

Reply:
Oh right. I created a `Helpers.swift` file. This is redundant, but the one that has the accuracy argument is needed. I'll remove this one.